target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -154,6 +154,12 @@ ipq40xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0u@eth0" "2:lan:4" "3:lan:3" "4:lan:2" "0u@eth1" "5:wan"
 		;;
+		
```

Comment:
unnecessary blank line.

Reply:
fixed, please refer to new PR: https://github.com/openwrt/openwrt/pull/10708